manifests/example-config.yaml

```diff
@@ -114,3 +130,28 @@ data:
       # re-advertisement outside of the immediate autonomous system,
       # but people don't usually recognize its numerical value. :)
       no-export: 65535:65281
+    bfd-profiles:
+    # The unique name to associate the bfd profile to, and to reference it
+    # from a peer entry. BFD is only available in FRR mode.
+    - name: bfdprofilename
+      # (optional) the minimum interval that this system is capable of 
+      # receiving control packets in milliseconds. The default value is 300 milliseconds
+      receive-interval: 60
```

Comment:
60 milliseconds is too demanding to be in an example manifest. Consider a value closer or higher to the default one.